### PR TITLE
vue3: Added phone number cell

### PR DESCRIPTION
**Pull request by mustafapc19**
Ref: #2922 

This is just for the time being, a `Text` cell as it is in `nc-gui` v1.

UI:

image

image

### CODE REVIEW

File: packages/nc-gui-v2/components/cell/PhoneNumber.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+})
+</script>
+
+<template>
+  <Text v-model="localState" />
+</template>
+
+<style scoped>
+input,
+textarea {
```
Comment (wingkwong) [inline packages/nc-gui-v2/components/cell/PhoneNumber.vue:25]: Use windicss. `@apply ...`

File: packages/nc-gui-v2/components/cell/PhoneNumber.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+interface Props {
+  modelValue: any
+}
+
+const { modelValue: value } = defineProps<Props>()
+
+const emit = defineEmits(['update:modelValue'])
+
+const localState = computed({
```
Comment (wingkwong) [inline packages/nc-gui-v2/components/cell/PhoneNumber.vue:13]: Try `useVModel`?

**Review by wingkwong — CHANGES_REQUESTED** [review]
Please also include an icon for `CellPhoneNumber` in `components/smartsheet-header/CellIcon.vue`. In v1, we're using `mdi:file-phone`. 

See https://icones.js.org/collection/mdi?s=mdi-file-phone

**Comment by mustafapc19** [comment]
@wingkwong Have added the icon

**Comment by wingkwong** [comment]
Please rebase, resolve conflicts & run lint.

File: packages/nc-gui-v2/components/cell/PhoneNumber.vue [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import Text from './Text.vue'
+
+interface Props {
+  modelValue: any
+}
+
+interface Emits {
+  (event: 'update:modelValue', model: number): void
```
Comment (wingkwong) [inline packages/nc-gui-v2/components/cell/PhoneNumber.vue:9]: should string be also allowed? e.g `+15614390540`

**Review by wingkwong — APPROVED** [review]

**Pull request merged**